cfme/tests/infrastructure/test_rest_automation_request.py

```diff
@@ -64,7 +64,19 @@ def create_requests(collection, rest_api, automation_requests_data, multiple):
 
 
 def create_pending_requests(collection, rest_api, requests_pending):
+    # Wait to see that the `approval_state` does NOT change.
+    # It's enough to wait just for the first request, it gives
+    # other requests the same amount of time to change state.
+    waiting_request = requests_pending[0]
+    wait_for(
+        lambda: waiting_request.approval_state != 'pending_approval',
```

Comment:
It checks that the state does NOT change, i.e. the state is still "pending_approval" after wait period. I'm expecting the `wait_for` to fail. Then for each request there's an assert that the "approval_state" is still "pending_approval".
The purpose is to wait for 30 sec and see if any request would auto-approve. I'll try to make the comment more clear.